dev/test_data.json

```diff
@@ -61,11 +61,23 @@
             "date": "12/05/2024",
             "index": 1
         },
+        {
+            "initials": "PSM",
+            "score": 312442,
+            "date": "12/05/2024",
+            "index": 2
+        },
+        {
+            "initials": "PSM",
+            "score": 312443,
+            "date": "12/05/2024",
+            "index": 3
+        },
         {
             "initials": "BBB",
             "score": 182398,
             "date": "11/05/2023",
-            "index": 2
+            "index": 4
```

Comment:
I wouldn't object if you wanted to put your initials in the test data. :shrug: :laughing: